lib/ukintctlr/arm/intctlr.c

```diff
@@ -30,7 +30,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 #include <uk/assert.h>
-#include <kvm/intctrl.h>
+#include <uk/intctlr.h>
 #include <arm/cpu.h>
 #include <arm/irq.h>
 #include <gic/gic.h>
```

Comment:
This needs to be `#include <uk/intctlr/gic.h>`

Reply:
Should't it be `#include <uk/gic/gic.h>`?